wikibase/1.30/bundle/Dockerfile

```diff
@@ -4,9 +4,21 @@ RUN apt-get update && \
     apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
 ADD download-extension.sh .
 RUN bash download-extension.sh OAuth
-RUN tar xzf OAuth.tar.gz && rm *.tar.gz
+RUN bash download-extension.sh Elastica
+RUN bash download-extension.sh CirrusSearch
+RUN tar xzf OAuth.tar.gz
+RUN tar xzf Elastica.tar.gz
+RUN tar xzf CirrusSearch.tar.gz && rm *.tar.gz
+
+FROM composer as composer
+COPY --from=fetcher /Elastica /Elastica
+RUN cd /Elastica && composer install --no-dev
 
 FROM wikibase/wikibase:1.30
 COPY --from=fetcher /OAuth /var/www/html/extensions/OAuth
+COPY --from=composer /Elastica /var/www/html/extensions/Elastica
+COPY --from=fetcher /CirrusSearch /var/www/html/extensions/CirrusSearch
 COPY LocalSettings.php.wikibase-bundle.template /LocalSettings.php.wikibase-bundle.template
+ADD extra-install.sh /
```

Comment:
Right now this looks like we have added something into the base entry point that then only every runs these extra files?
If this is meant to be specifically use for the bundle image lets call them bundle-install.sh etc instead?

If this is meant to be more generic then I guess we want a directory that .sh files can be copied into instead? Or some way things can be appended to files? But neither of those seem worth it just yet.
I think naming this budle etc would be nicer.